### PR TITLE
Lcra/change security visibility per role prod 14083

### PR DESCRIPTION
**Pull request by Leopold-Cramer**

### CODE REVIEW

**Review by jreynard-code — APPROVED** [review]
LGTM

File: solution/src/integrationTest/kotlin/com/cosmotech/solution/service/SolutionServiceIntegrationTest.kt [hunk 1]
```diff
@@ -379,6 +380,102 @@ class SolutionServiceIntegrationTest : CsmRedisTestBase() {
...
+              organizationSaved.id!!,
+              baseSolutionSaved.id!!,
+              "WrongRunTemplateId",
+              RunTemplate(id = "FakeRunTemplateId"))
+        }
+    assertEquals("Run Template 'WrongRunTemplateId' *not* found", assertThrows.message)
+  }
+
+  @Test
+  fun `test update solution with runTemplates specified`() {
```
Comment (sellisd) [inline solution/src/integrationTest/kotlin/com/cosmotech/solution/service/SolutionServiceIntegrationTest.kt:426]: failing test

File: solution/src/integrationTest/kotlin/com/cosmotech/solution/service/SolutionServiceIntegrationTest.kt [hunk 2]
```diff
@@ -379,6 +380,102 @@ class SolutionServiceIntegrationTest : CsmRedisTestBase() {
...
+        solutionApiService.updateSolution(
+            organizationSaved.id!!,
+            baseSolutionSaved.id!!,
+            baseSolutionSaved.apply { runTemplates = modifiedSolutionRunTemplates })
+
+    assertEquals(baseSolutionRunTemplates, updateSolutionSaved.runTemplates)
+  }
+
+  @Test
+  fun `test update solution with empty runTemplates specified`() {
```
Comment (sellisd) [inline solution/src/integrationTest/kotlin/com/cosmotech/solution/service/SolutionServiceIntegrationTest.kt:457]: failing test

**Review by sellisd — APPROVED** [review]
after applying PROD-13734 fix and correcting versions seems to me ok for merging

**Pull request merged**